harpy/assembly.py

```
@@ -11,90 +12,133 @@
+    
     create_conda_recipes()
     if setup_only:
         sys.exit(0)
 
     start_text = Table(show_header=False,pad_edge=False, show_edge=False, padding = (0,0), box=box.SIMPLE)
     start_text.add_column("detail", justify="left", style="light_steel_blue", no_wrap=True)
     start_text.add_column("value", justify="left")
+    start_text.add_row("Metassembly: ", "True" if metassembly else "False")  
     start_text.add_row("Barcode Tag: ", bx_tag.upper())
     if kmer_length == "auto":
         start_text.add_row(f"Kmer Length: ", "auto")
```

Ruff

f-string without any placeholders

Remove extraneous `f` prefix

(F541)